### DISCUSSION TITLE
Goroutines are steadily rising

### DISCUSSION THREAD

**Discussion by grinapo** [navidrome/navidrome#2942] [General]
Is it normal that number of (active?) **goroutines** (metric `go_goroutines`) is steadily rising by about 50-70 per hour, even when the server does nothing really special (no new files, no playback)?

**Comment by dweymouth** [comment]
Is it possible that that metric is counting the total number of spawned goroutines (active + exited)? If so, it would be expected for it to increase like that. If it is actually a count of **active** goroutines then there is a problem as some goroutines are probably not exiting like they should.
